travis.sh

```diff
@@ -1,27 +0,0 @@
-#!/bin/bash
```

Comment:
And it'd be good to add a comment header to simply state that this file remains as a backward compatibility; if I were a new user of this package I would have been tempted to simply remove this.